tests/006-test-overlay-cleanup.sh

```diff
@@ -23,9 +23,11 @@ EOF
  if [ $? -ne 0 ]; then
     # Test failed.
     test_status=1
+    echo $DSSBIN --reset "Failed" >> $LOGS 2>&1
  elif [ -e /etc/sysconfig/docker-storage ]; then
     # Test failed.
     test_status=1
+    echo "$DSSBIN --reset Failed: /etc/sysconfig/docker-storage still exist" >> $LOGS 2>&1
  fi
   if [ ${test_status} -eq 1 ]; then
     echo "ERROR: $testname: $DSSBIN --reset Failed." >> $LOGS
```

Comment:
Given you have added specific message for errors. may be lets change this message to just inform that test failed. Anyways this message is little misleading now.

`ERROR: $testname Failed`